Source/WebCore/rendering/MotionPath.cpp

```diff
@@ -32,11 +32,10 @@
 
 namespace WebCore {
 
-static FloatPoint offsetFromContainer(const RenderObject& renderer, const FloatRect& referenceRect)
+static FloatPoint offsetFromContainer(const RenderObject& renderer, const FloatRect& )
```

Comment:
Let's remove this second param now that it's unused

Reply:
Fixed.